docs/public/deploying-a-devfile-using-odo.adoc

```diff
@@ -56,7 +53,17 @@ organized in a separate single unit.
    ✓  Project 'myproject' is ready for use
    ✓  New project created and now using project : myproject
 ----
+
+
+==== Prerequisites for a Kubernetes Cluster
+
+* Enable experimental mode for odo. This can be done by: `odo preference set experimental true`
+
+* Before proceeding, you must know your ingress domain name or ingress ip to specify `--host` for `odo url create`.
 +
+Ingress ip is usually the external ip of ingress controller service, for minikube or crc clusters running in a virtual machine you can get it by
```

Comment:
I feel like this can be shortened? 

IP needs to be capitalized. Same for CRC and Minikube.

I feel as though since people are developing against Kubernetes and OpenShift already, they already know what an Ingress is. So we can remove that part. The part about minikube or crc looks good though.

Reply:
@cdrage its just one line, just added to redirect on some ingress document as @kadel suggested above